packages/apm/src/span.ts

```diff
@@ -261,7 +272,9 @@ export class Span implements SpanInterface, SpanContext {
       return undefined;
     }
 
-    this.timestamp = timestampWithMs();
+    // TODO: Fallback to timestampWithMs() when performance.now is unavailable.
```

Comment:
Suggestions for how to do this welcome.

Could use a polyfill for `performance.now` or conditionally set `_startTimestampMonotonic` to some sentinel value and check it here.

Reply:
It looks like the policy for the JS SDK is to not polyfill anything older than Edge (i.e. IE 11 and older): https://github.com/getsentry/sentry-javascript/issues/2003

So, I think we should be safe in not worrying about this 👍